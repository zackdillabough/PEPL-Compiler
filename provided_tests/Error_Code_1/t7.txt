POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F1(A, B) = A - A + B;

POLY G = x^2 + x;

POLY F3 = x;
POLY F4 = x + 1;
POLY F2(X) = X^5050 - 38792647;
POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F1(A, B) = A - A + B;
POLY F3 = x;
POLY F4 = x + 1;
POLY F2(X) = X^5050 - 38792647;
POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;

POLY G1(T) = T + 2 T + 5 T^3;

POLY F1(A, B) = A - A + B;
POLY F3 = x;
POLY F4 = x + 1;
POLY F2(X) = X^5050 - 38792647;
POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F1(A, B) = A - A + B;
POLY F3 = x;
POLY F4 = x + 1;
POLY F2(X) = X^5050 - 38792647;

POLY G2(A, S, D, F, G, H, J, K, L) = 9;


POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F1
	(A, B) = A - A + B;
POLY F3 = x;
POLY F4 = x + 1;
POLY F2
 	(X) = X^5050 - 38792647;


POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F1

	(A, B) = A - A + B;
POLY F3 = x;
POLY F4 = x + 1;
POLY F2(X) = X^5050 - 38792647;
POLY 

	F1(X) 
		= X + 1;
POLY F2 = x^2 + x + 1;
POLY F1(A, B) = A - A + B;
POLY F3
	 = x;
POLY F4 = x + 1;
POLY F2
		(X) = X^5050 - 38792647;








POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F1(A, B) = A - A + B;
POLY F3 = x;
POLY F4 = x + 1;
POLY F2(X) = X^5050 - 38792647;
START
INPUT X;
INPUT Y;

G1(Y);
G(X);

INPUT X;
INPUT Y;

G1(Y);
G(X);

INPUT X;
INPUT Y;

G1(Y);
G(X);

34 21 4 8 72 1 2778 343 1386
