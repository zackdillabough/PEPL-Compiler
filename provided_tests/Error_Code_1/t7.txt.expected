Error Code 1: 1 2 3 7 8 9 10 11 12 13 14 15 16 17 21 22 23 24 25 26 27 28 29 30 35 36 37 39 40 41 45 46 47 50 51 52 55 57 58 59 61 62 72 73 74 75 76 77 